extras/rya.jena.sesame/NOTICE

```diff
@@ -0,0 +1,11 @@
+Apache Jena - ARQ module
+Copyright 2011, 2012 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+Portions of this software were originally based on the following:
+  - Copyright 2004, 2005, 2006, 2007, 2008, 2009 Hewlett-Packard Development Company, LP
+  - Copyright 2010, 2011 Epimorphics Ltd.
+  - Copyright 2010 Talis Systems Ltd.
+These have been licensed to the Apache Software Foundation under a software grant.
```

Comment:
Not sure why this is here. The top-level NOTICE file covers the source-release. You *do* need to make sure that the rya.jena.sesame's JAR file does contain the correct `META-INF/{LICENSE,NOTICE}` files.